presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestUtilizedColumnsAnalyzer.java

```diff
@@ -45,6 +45,13 @@ public void testWildcardSelect()
                 ImmutableMap.of(QualifiedObjectName.valueOf("tpch.s1.t1"), ImmutableSet.of("a", "b")));
     }
 
+    @Test
+    public void testSubField()
```

Comment:
quick nits: let's use word "Subfield" with lower case f. Because we have the term subfield used in other places as well. For example `PushdownSubfields`. So just to make it consistent